.circleci/config.yml

```diff
@@ -224,6 +235,36 @@ jobs:
           working_directory: examples/default
           command: detox test -c android.emu.release
 
+  # Automate the enterprise NN sdk changes
+  release_nn:
+    working_directory: ~/project
+    macos:
+      xcode: 13.4.1
```

Comment:
Do we need to run it on a macOS machine? If not, I think we can switch to a `node/default` executor instead since it's usually faster and cheaper.
I believe switching to `node/default` which runs on Linux would also break the current `sed` command and we might need to remove the `"" -e` part of the command so let's make sure everything is still working as expected.

Reply:
AFAIK the `sed` command is a bit different on macOS as you need to add `""` which isn't needed on Linux, we just need to make sure that we update the job .